config/ingress/controller.yaml

```diff
@@ -64,6 +64,8 @@ spec:
           value: config-observability
         - name: METRICS_DOMAIN
           value: knative.dev/samples
+        - name: INGRESS_CLASS_NAME
+          value: istio.ingress.networking.knative.dev
```

Comment:
Can you please document in README what the different possible values are?

Reply:
README now covers how to change the prefix of the ingress, with examples given and flexibility to bring your own.